### ISSUE TITLE
Gruntfile Syntax Error

### ISSUE THREAD

**Issue by ZaLiTHkA** [gilbarbara/generator-web#2]
Just a quick note: I found a small error in the Gruntfile just now... Lines [181 and 183](https://github.com/gilbarbara/generator-web/blob/master/app/templates/Gruntfile.js#L181-L183) have commas in the wrong place:

```
            'less'
            'jshint',
            'uglify',,<% if (includeModernizr) { %>
```

**Comment by gilbarbara** [comment]
fixed in 7e37339f6d011df72cd02520f43a7adc77515cf9
Thanks

**Comment by ZaLiTHkA** [comment]
@gilbarbara. My apologies, only just noticed your message. Thanks for the quick fix.. :)
